packages/nc-gui/components/account/Token.vue

```diff
@@ -12,22 +12,44 @@ const { copy } = useCopy()
 
 const { t } = useI18n()
 
-const tokens = ref<UserType[]>([])
+interface IApiTokenInfo extends ApiTokenType {
+  created_by: string
+}
+
+const tokens = ref<IApiTokenInfo[]>([])
+
+const selectedToken = reactive({
+  isShow: false,
+  id: '',
+})
 
 const currentPage = ref(1)
 
 const showNewTokenModal = ref(false)
 
 const currentLimit = ref(10)
 
-const selectedTokenData = ref<ApiTokenType>({})
+const selectedTokenData = ref<ApiTokenType>({
+  description: 'untitled token',
```

Comment:
`untitled token` occurs 3 times. Use a variable instead so that you just need to update once instead of thrice in case there is a change. Also apply i18n if possible.